Source/WebKit/WebProcess/Extensions/API/Cocoa/WebExtensionAPINamespaceCocoa.mm

```diff
@@ -49,7 +49,10 @@
 
 #if ENABLE(INSPECTOR_EXTENSIONS)
     if (name == "devtools"_s)
-        return objectForKey<NSDictionary>(extensionContext().manifest(), @"devtools_page");
+        return objectForKey<NSString>(extensionContext().manifest(), @"devtools_page") && (page.isInspectorPage() || extensionContext().isInspectorBackgroundPage(page));
```

Comment:
Can we share "devtools_page" with the definition in `WebExtensionCocoa.mm`?

Reply:
Not easily. We use less static strings like that is WebKit when compared to Safari.